index.html

```diff
@@ -186,6 +186,25 @@ <h1>Proof Methods</h1>
   </thead>
 
   <tbody>
+    <tr>
+      <td>
+          Ed25519Signature2020
+      </td>
+      <td>
+          <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">
+            Ed25519 Signature 2020</a>
+      </td>
+      <td>
+          Orie Steele
+      </td>
+      <td>
+        <a href="https://github.com/digitalbazaar/ed25519-signature-2020">Implemented</a>
```

Comment:
Does this mean this is a reference implementation?

Should other registered proof/extension types also link to one or more implementations?

Reply:
It's meant to convey that it is *an* implementation, not necessarily *the* implementation. We try to stay away from saying "reference" implementation because sometimes multiple people think they are creating the reference implementation. :)

Is there a better way this could be conveyed?

Reply:
That makes sense, thanks.

I think an implementations column could work, or having multiple implementations listed in some way, that could make it more clear.

I'm okay with letting future PRs figure it out (how to list more than one implementation); this PR already has approvals as-is.

Reply:
Ok, then I'll merge this one and raise another one to add that column.

Reply:
Raised PR https://github.com/w3c-ccg/vc-extension-registry/pull/12 to address @clehner's comment above. It turned into a larger refactor, but one that I hope the community will accept.